eai25_experiments/experiment.py

```
@@ -0,0 +1,83 @@
+    final_answer = extract_final_answer_from_response(response.choices[0].message.content)
+    if final_answer is None:
+        raise ValueError(f"Invalid response '{final_answer}'.")
+    return SCORE_MAP[final_answer]
+
+
+def run_experiment(n_gpt_samples_per_task: int = 10) -> pd.DataFrame:
+    annotations_df = pd.read_csv(os.path.join(DATA_DIR, "annotations.csv"))
+    screenshot_subjects = list(annotations_df.columns)[1:]  # 1st col = annatator_name
+    tasks = [f"Take a screenshot of {subject}" for subject in screenshot_subjects]
+    scores: dict[str, list[float]] = defaultdict(list)
+    for task, subject in zip(tasks, screenshot_subjects):
```

Ruff (0.8.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)